app/models/broker_stock.rb

```diff
@@ -1,2 +1,7 @@
 class BrokerStock < ApplicationRecord
+  validates :user_id, presence: true
+  validates :stock_id, presence: true
```

Comment:
No need to validates foreign_key, just define the database association of `BrokerStock` with `User` and `Stock` model

Reply:
Noted on this Ma'am. Will update the model po. Thank you.